packages/back-end/src/services/experiments.ts

```diff
@@ -1972,6 +1972,13 @@ export function visualChangesetsHaveChanges({
     return true;
   }
 
+  // If there are redirect url differences
+  const oldRedirects = oldVisualChangeset.urlRedirects?.map((r) => r.url);
+  const newRedirects = newVisualChangeset.urlRedirects?.map((r) => r.url);
+  if (!isEqual(oldRedirects, newRedirects)) {
+    return true;
+  }
```

Comment:
Should we also add a check to see if `persistQueryString` has changed?

Reply:
Yes we should. I’ll add that check in